### PR TITLE
[risk=low][RW-4838] Rename a few "DUA" to "DUCC"

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

This is only a small portion of RW-4838.  Most of the renaming will occur as as part of the migration to the new Access Modules system.

This includes a small change to Workspace Edit:
Screen Shot 2021-07-14 at 2 41 49 PM

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 1]
```diff
@@ -221,8 +221,8 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
       misused by others to foster stigma. While it may not be possible to completely prevent misuse
       of research for stigmatizing purposes, data users can take important steps to minimize the
         risk of this happening–</div>
-      <TooltipTrigger content={toolTipTextDataUseAgreement}>
-        <div>taking this step is a condition of your Data Use Agreement.</div>
+      <TooltipTrigger content={toolTipTextDucc}>
+        <div>taking this step is a condition of your Data User Code of Conduct agreement.</div>
```
Comment (jmthibault79) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:225]: this is the only user-visible change

File: ui/src/app/pages/workspace/workspace-edit.tsx [hunk 2]
```diff
@@ -1372,9 +1372,8 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
             in findings that may be misinterpreted or misused by others to foster stigma. While it
             may not be possible to completely prevent misuse of research for stigmatizing purposes,
             data users can take important steps to minimize the risk of this happening–
-            <TooltipTrigger content={toolTipTextDataUseAgreement}>
-              <Link style={{display: 'inline-block'}}>taking this step is a condition of your
-                Data Use Agreement.</Link>
+            <TooltipTrigger content={toolTipTextDucc}>
+              <Link style={{display: 'inline-block'}}>taking this step is a condition of your Data User Code of Conduct agreement.</Link>
```
Comment (jmthibault79) [inline ui/src/app/pages/workspace/workspace-edit.tsx:1376]: this is the only user-visible change

**Comment by jmthibault79** [comment]
e2e failure is "Datasets card snowman menu actions Rename dataset" which is unlikely to be related.  Previous runs have passed.

**Review by petesantos — APPROVED** [review]
LGTM

**Pull request merged**